assets/js/hooks/comment_input.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * A hook for the comment input.
+ *
+ * On submit this hook clears the text input, which is not done
+ * automatically when using keyboard shortcuts ('Enter' key).
+ */
+const CommentInput = {
```

Comment:
We can most likely do everything we need with JS hooks and custom events :) Here are some event handles we already have:

https://github.com/livebook-dev/livebook/blob/d567adaa173acfb0b133bb83a94662ccf14318bc/assets/js/events.js#L25-L36

This way you can use `JS.dispatch` to focus/clear the input.

Reply:
done in b2d2f4a